### PR TITLE
chore: update ThreeJS to 0.122.0

### PR DESCRIPTION
**Pull request by larsmoa**

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "styled-components": "^5.2.0",
-    "three": "link:../viewer/node_modules/three"
+    "three": "^0.122.0"
```
Comment (maksnester) [inline examples/package.json:39]: @larsmoa why to remove the link? We just will run into conflicts between versions as we had before.

**Review by maksnester — CHANGES_REQUESTED** [review]
https://github.com/cognitedata/reveal/pull/1285/files#r514138383

cc @larsmoa

File: examples/package.json (hunk 1 above)
Comment (larsmoa) [reply examples/package.json:39]: @maksnester : oops, should we have a similar link in documentation btw?

Comment (maksnester) [reply examples/package.json]: Sure, we link reveal and sdk already. Probably make sense to do the same for three.js too.

**Review by maksnester — APPROVED** [review]

**Pull request merged**